views/page/course/module/events/cancel-attendee.html

```diff
@@ -0,0 +1,103 @@
+{% extends "../../../../component/Page.njk" %}
+
+{% from "select/macro.njk" import govukSelect %}
+{% from "button/macro.njk" import govukButton %}
+
+{% block pageTitle %}{{ i18n['cancel_attendee'] }} - {{ i18n['proposition_name'] }}{% endblock %}
```

Comment:
I have no `cancel_attendee` entry in en.json? Also can you please use the object structure in en.json for events/attendees, similar to what we've got in for `module` and `termsAndConditions`